projects/scripts/adi_project_xilinx.tcl

```diff
@@ -280,7 +280,20 @@ proc adi_project_run {project_name} {
       } issue ] != 0 } {
         puts "GENERATE_REPORTS: tclapp::xilinx::designutils not installed"
       }
-  } else {
+
+      # Define a list of IPs for which to generate report utilization
+      set IP_list { ad_ip_jesd_204_tpl_adc ad_ip_jesd_204_tpl_dac axi_jesd204_rx axi_jesd204_tx jesd204_rx jesd204_tx axi_adxcvr util_adxcvr axi_dmac util_cpack2 util_upack2 }
```

Comment:
please could you split this line on multiple lines so it can be easier reviewed/maintained.

Reply:
Done